code/modules/admin/topic.dm

```diff
@@ -1836,9 +1847,53 @@
 				evilcookie.reagents.add_reagent("hell_water", 25)
 				evilcookie.desc = "Sulphur-flavored."
 				evilcookie.bitesize = 100
+				evilcookie.flags = NODROP
 				H.drop_l_hand()
 				H.equip_to_slot_or_del(evilcookie, slot_l_hand)
 				logmsg = "a hellwater cookie."
+			if("Assassin")
+				logmsg = "assassin."
+				var/list/possible_traitors = list()
+				for(var/mob/living/player in mob_list)

```

Comment:
don't loop through the entire mob list when you can loop through the living mob's list, which is faster.